src/optimization/hooke_jeeves_method.cpp

```diff
@@ -0,0 +1,149 @@
+#include <k52/optimization/hooke_jeeves_method.h>
+
+#ifdef BUILD_WITH_MPI
+
+#include <boost/mpi.hpp>
+#include <k52/parallel/mpi/constants.h>
+
+#endif
+
+#include <iostream>
+#include <cmath>
+#include <algorithm>
+#include <stdexcept>
+
+#include <k52/optimization/params/i_continuous_parameters.h>
+
+using ::std::vector;
+
+namespace
+{
+	double kAcceleration = 3.5;
+}
+
+namespace k52
+{
+namespace optimization
+{
+
+HookeJeevesMethod::HookeJeevesMethod(double precision, size_t max_iteration_number,
+	double init_step)
+	: precision_(precision)
```

Comment:
I'm not sure that I've got the point. Is it so critical? Should I initialize variables in alphabetical order or some other way?